site/content/en/docs/drivers/hyperv.md

```diff
@@ -6,15 +6,17 @@ aliases:
 ---
 ## Overview
 
-Hyper-V is a native hypervisor built in to modern versions of Microsoft Windows.
+[Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/) is a native hypervisor built in to modern versions of Microsoft Windows.
 
 {{% readfile file="/docs/drivers/includes/hyperv_usage.inc" %}}
 
 ## Special features
 
 The `minikube start` command supports additional hyperv specific flags:
 
-* **`--hyperv-virtual-switch`**: The hyperv virtual switch name. Defaults to first found
+* **`--hyperv-virtual-switch`**: Name of the virtual switch the minikube VM should use. Defaults to first found
+* **`--hyperv-use-external-switch`**: Use external virtual switch over Default Switch if virtual switch not explicitly specified, creates a new one if not found. If the adapter is not specified, the driver first looks up LAN adapters before other adapters (WiFi, ...). Or the user may specify an adapter to attach to the external switch. Default false
+* **`--hyperv-external-adapter`**:  External adapter on which the new external switch is created if no existing external switch is found. Since Windows 10 only allows one external switch for the same adapter, it find the virtual switch before creating one. The external switch is created and named "minikube" if not the case
```

Comment:
nit: "it finds the virtual switch" instead of "it find"
also, I'd remove "if not the case" at the end

Reply:
Ok, will fix it.

Reply:
Done